File: fizzbuzz.py
def fizzbuzz():
    outputArr = []
    n = 30
    for value in range(1, n+1):
        if value % 3 == 0 and value % 5 == 0:
            outputArr.append("FizzBuzz")
        elif value % 3 == 0:
            outputArr.append("Fizz")
        elif value % 5 == 0:
            outputArr.append("Buzz")
        else:
            outputArr.append(value)
    print(outputArr)


fizzbuzz()

File: fizzbuzz.go
package main

import (
	"fmt"
	"strconv"
)

func main() {
	n := 30
	fizzBuzz(n)
}

func fizzBuzz(A int) []string {
	outputArr := []string{}

	for i := 1; i <= A; i++ {
		if i%5 == 0 && i%3 == 0 {
			outputArr = append(outputArr, "FizzBuzz")
		} else if i%5 == 0 {
			outputArr = append(outputArr, "Buzz")
		} else if i%3 == 0 {
			outputArr = append(outputArr, "Fizz")
		} else {
			outputArr = append(outputArr, strconv.Itoa(i))
		}
	}
	fmt.Println(outputArr)
	return outputArr
}
